captum/attr/_core/layer/layer_integrated_gradients.py

```diff
@@ -258,12 +272,19 @@ def attribute(
 
         # inputs -> these inputs are scaled
         def gradient_func(
-            forward_fn, inputs, target_ind=None, additional_forward_args=None
-        ):
+            forward_fn: Callable,
+            inputs: Union[Tensor, Tuple[Tensor, ...]],
+            target_ind: int = None,
```

Comment:
This should actually be the same type as target above, (Optional[Union[int, Tuple[int, ...], Tensor, List[Tuple[int, ...]]]])

Reply:
Good catch!  Thanks, fixed.